pkg/apis/certmanager/v1/generic_issuer.go

```diff
@@ -35,43 +36,67 @@ type GenericIssuer interface {
 var _ GenericIssuer = &Issuer{}
 var _ GenericIssuer = &ClusterIssuer{}
 
+// GetObjectMeta is ...
```

Comment:
Is this a reasonable pattern?
```suggestion
// GetObjectMeta is an accessor for `ObjectMeta`
```